x-pack/plugins/security_solution/public/common/components/exceptions/helpers.tsx

```diff
@@ -440,7 +440,7 @@ export const defaultEndpointExceptionItems = (
           ],
         },
         {
-          field: 'file.path',
+          field: 'file.path.text',
```

Comment:
We agreed here: https://github.com/elastic/endpoint-dev/issues/6938#issuecomment-626906011 that we would do case insensitive for EVERYTHING other than signer. Couldn't we just underneath the hood at a ".text" to the exceptionlist entry as we create it and hide that from the user?  (For everything other than signer)? Exposing the .text to the user will maybe be confusing to most users?

Reply:
With that approach endpoint exceptions on fields that don't have a `.text` mapping (e.g. `file.name`) won't work on the detection engine side. I'm not sure how important it is to have endpoint exceptions execute in both the endpoint and detection engine.